src/components/PostcodeLookup/PostcodeLookup.js

```diff
@@ -359,8 +359,9 @@ class PostcodeLookup extends Component {
       invalidErrorText: 'Please enter a valid postcode',
       emptyFieldErrorText: 'Please enter your postcode',
       extraClass: 'search-box',
+      autocomplete: 'off',
     };
-    const addressPattern = /^[A-Za-z0-9_.'/&\s-]+$/;
+    const addressPattern = /^[A-Za-z0-9][A-Za-z0-9_.'/&\s-]+$/;
```

Comment:
Tested it on the preview environment and I think I made a mistake when giving you this regex pattern. 
This will show the error message when you type in the first allowed character.
I think this pattern might be better on second thought:
`^[a-zA-Z0-9]+[-.'/&\s\w]*$`
Starting of with needing one or more characters in the first set and then zero or more in the second set.
`\w` is also allowing an underscore, so that's why I didn't use it in the first set.